WordPress/Resources/ar.lproj/Localizable.strings

```diff
@@ -7035,6 +7094,18 @@ translators: Block name. %s: The localized block name */
 /* Message shown when the reader finds no posts for the chosen site */
 "This site has not posted anything yet. Try back later." = "لم ينشر هذا الموقع أي شيء بعد. حاول مرة أخرى لاحقًا.";
 
+/* Error message displayed when unable to close user account due to unresolved chargebacks. */
+"This user account cannot be closed if there are unresolved chargebacks." = "This user account cannot be closed if there are unresolved chargebacks.";
+
+/* Error message displayed when unable to close user account due to having active atomic site. */
+"This user account cannot be closed while it has active atomic sites." = "This user account cannot be closed while it has active atomic sites.";
```

Comment:
Notice that this and all the other non-English US localization of this string, match the original version not the new one updated from the change in `WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift`.

That's expected because these files were downloaded GlotPress during the `download_localized_strings_and_metadata` step of the `new_beta_release`. Because the change to this string hasn't been merged into `develop` yet, GlotPress still has the original version.

Reply:
👌